Sources/GuakaClILib/MiscTypes.swift

```diff
@@ -20,35 +22,53 @@ public enum GuakaError: Error {
   case cannotReadFile(String)
   case setupFileAltered
   case notAGuakaProject
+  case commandAlreadyExist(String, String)
   case missingCommandName
   case tooManyArgsPassed
   case wrongCommandNameFormat(String)
 
   public var error: String {
     switch self {
     case .wrongDirectoryGiven(let path):
-      return "The path given cannot be used \(path)"
+      return [
+        "Wrong path given:",
+        "  \(path)",
+        "The path cannot be used as its not empty"
+        ].joined(separator: "\n").f.red
+
     case .triedToCreateProjectInNonEmptyDirectory(let path):
-      return "Cannot create project in non empty directory \(path)\n"
+      return "Cannot create project in non empty directory: \(path)\n".f.red
+
     case .failedCreatingFolder(let path):
-      return "Failed creating directory \(path)"
+      return "Failed creating directory \(path)".f.red
+
     case .cannotCreateFile (let name):
-      return "Cannot generate \(name) file"
+      return "Cannot generate \(name) file".f.red
+
     case .cannotReadFile(let path):
-      return "Cannot read contents of file \(path)"
+      return "Cannot read contents of file \(path)".f.red
+
     case .setupFileAltered:
-      return "Guaka setup.swift file has been altered.\nThe placeholder used to insert commands cannot be found \(GeneratorParts.comamndAddingPlaceholder).\nYou can try to add it yourself by updating `setup.swift` to look like\n\n\(GeneratorParts.setupFileContent())\n\nAdding command wont be possible."
+      return "Guaka setup.swift file has been altered.\nThe placeholder used to insert commands cannot be found \(GeneratorParts.comamndAddingPlaceholder).\nYou can try to add it yourself by updating `setup.swift` to look like\n\n\(GeneratorParts.setupFileContent())\n\nAdding command wont be possible.".f.red
+
     case .notAGuakaProject:
-      return "This command can only be executed in a Guaka project.\nThe following directory does not contain guaka files"
+      return "This command can only be executed in a Guaka project.\nThe following directory does not contain guaka files".f.red
```

Comment:
"This command can only be executed inside a Guaka project." I think it's enough.